src/main/java/com/github/maxopoly/MemeMana/MemeManaIdentity.java

```diff
@@ -0,0 +1,54 @@
+package com.github.maxopoly.MemeMana;
+
+import com.github.maxopoly.MemeMana.model.MemeManaOwner;
+import java.util.Set;
+import java.util.UUID;
+import vg.civcraft.mc.namelayer.NameAPI;
+import com.programmerdan.minecraft.banstick.data.BSPlayer;
+import org.bukkit.entity.Player;
+
+public class MemeManaIdentity implements MemeManaOwner {
```

Comment:
This entire class shouldnt exist imo, instead BanStick/AltManager should offer a cleaner API, like a single int per alt group.

Reply:
I think this class should still exist later on, but strictly to be either and AltManager/BanStick altgroup Id *OR* a NameLayer group instead. My plan is to move most of the code dealing with this stuff to AltManager anyway, just didn't want to have to deal with compiling multiple projects with snapshot versions etc.

Reply:
Yeah okay. A player specific implementation of MemeManaOwner is fine (and intended), I mostly meant that all the alts stuff is out of place here. If you already have plans for moving it, perfect.